property-service/src/main/java/org/egov/ps/model/Document.java

```diff
@@ -37,6 +37,10 @@
 	@JsonProperty("referenceId")
 	private String referenceId;
 
+	@EqualsAndHashCode.Include
```

Comment:
Why do we need a duplicate property `code` which maps to `documentType`. We already have a `documentType`?

Reply:
@rajiv-transerve ...

Document Validation as I understood ....
 condition 1 : owner document list must be same as master JSON that means documentList.code 
 condition 2 : owner document type must be match with master JSON that means documentList.accept

validation base on code and document extension ""accept": "application/msword,application/pdf,application/vnd.openxmlformats-officedocument.wordprocessingml.document,image/*","